Colours.txt

```diff
@@ -1,76 +1,76 @@
-squares
```

Comment:
Could you explain the changes in this file (and Config.txt)? I see you've replaced line endings to use Windows-style breaks, but why was this necessary?

Reply:
No, it was not necessary. I even have no idea how all these cr/lf came to files (I'm in Linux). Moreover, all config files (Config.txt, Colours.txt and hiscores.dat) are optional. If there are no any, they will be created at the first run. So you can safely ignore these changes, or remove mentioned files from repository at all.